### ISSUE TITLE
keras.layers' has no attribute 'DepthwiseConv2D

### ISSUE THREAD

**Issue by SreenijaK** [qubvel/segmentation_models#303]
i get the following error 
AttributeError: module 'keras.layers' has no attribute 'DepthwiseConv2D'

when i run 
model = sm.Unet(BACKBONE, classes=n_classes, activation=activation);

while I'm running the binary segmentation (camvid).ipynb

**Comment by qubvel** [comment]
What is your keras version?

**Comment by SreenijaK** [comment]
Version: 2.0.8

**Comment by qubvel** [comment]
Update keras, please. Look at requirements in README

**Comment by SreenijaK** [comment]
yes thats the issue. My bad. I was going through the multiclass segmentation. i didnt understand where are we assigning the class color to the label. for example in my annotation file if i marked all the cars with red, where are we mapping the red color with cars?
